source/chapters/cookbook.rst

```diff
@@ -0,0 +1,55 @@
+.. include:: /shortcuts.rstext
+
+.. _cookbook:
+
+Cookbook (Examples)
+*******************
+
+The Cookbook is here to give Mixxx users examples to work off of. The recipes both cover common or interesting use cases, and progressively explain more complex detail. This chapter moves beyond simple introductory examples and demonstrates concepts that will help users create awesome music, as well as the approach to achieving this.
+Each recipe will focus on one specific aspect of Mixxx, combining different features in interesting ways to fully explore its niche.
+
+Deck Cloning
+============
+
+When deck cloning (also called "instant doubles" in some other DJ software) is activated, the currently loaded song, playback state, pitch adjustment, and any playing loop will be cloned from one deck to another. This opens up a number of creative mixing possibilities, some of which we will explore below.
+
+.. image:: ../_static/deckclone.gif
+    :align: center
+
+How to use deck cloning
+-----------------------
```

Comment:
I don't think this subsection heading adds anything. I think it can simply be deleted and the text below can just be part of the the larger "Deck Cloning" section.

Reply:
Then the rest of the subsections can be moved up one level from `~~~~` to `-----`

Reply:
Okay. Noted.